tests/html_handlebars.rs

```diff
@@ -0,0 +1,101 @@
+extern crate env_logger;
+extern crate futures;
+extern crate handlebars;
+extern crate http;
+#[macro_use]
+extern crate tower_web;
+
+#[macro_use]
+mod support;
+use support::*;
+
+use handlebars::Handlebars as Registry;
```

Comment:
Now I'm conflicted... should this be consistent?

Reply:
`handlebars::Handlebars` is actually a re-export of `Registry` internally :shrug: